### ISSUE TITLE
Account suspended for using bot

### ISSUE THREAD

**Issue by darrylcurry616** [Alkaar/resy-booking-bot#136]
Anyone getting suspended for using bot on resy?
And do anyone has any solution for account suspension for bot using?.

**Comment by jfach** [comment]
Yes, I just had an account suspended. I think it's part of the game at this point.

**Comment by tableforyounyc** [comment]
Stop using the bot, your account will not get suspended.
